test/e2e/label/label.go

```diff
@@ -0,0 +1,72 @@
+package label
+
+// The label package contains a list of labels that can be used in
+// the e2e and serial tests to indicate and point certain behavior or characteristics
+// of the various tests.
+// Those can be filtered/focused by ginkgo before the test runs.
+
+// Kind is a label indicates specific criteria that classify the test.
+// A Mixture of kinds can be used for the same test
+type Kind = string
+
+const (
+	// Slow means test that usually requires reboot or takes a long time to finish.
+	Slow Kind = "slow"
+
+	// ReleaseCritical is for tests that are critical for a successful release of the component that is being tested.
+	ReleaseCritical Kind = "release-critical"
+
+	// SpecializedHardware is for tests that need special Hardware like SR-IOV devices, specific NICs, etc.
+	SpecializedHardware Kind = "specialized-hardware"
+)
+
+// Tier is a label to classify tests under specific grade/level
+// that should roughly describe the execution complexity, maintainer identity and processing criteria.
+type Tier = string
+
+const (
+	// Tier0 are automated unit tests
+	// Minimal time needed to execute (minutes to 1 hour)
+	// Process criteria:
+	// 100% automated, must-pass 100%
+	// Development maintains tests, and reviews result
```

Comment:
descriptions are nice but they don't reflect what our serial tests are and wants to be

Reply:
Please check again i've updated the desc